test/test.hs

```diff
@@ -106,6 +107,134 @@ macroTests = testGroup "macroTests" $
+         @?= "xyyxyyxyyxyy"
+     , testCase "RepeatLast test 24" $
+         snd (last (unwindMacros (bindInput [("a", "'xy'vv")] coinput)
+                                 (stringToKeyMacro "'aa'vv")))
+         @?= "xyyyxyyyxyyyxyyy"
+     , testCase "RepeatLast test 25" $
+         snd (last (unwindMacros (bindInput [("a", "'xyv'v")] coinput)
+                                 (stringToKeyMacro "'a'a'vv'")))
+         @?= "xyyyxyyyxyyyxyyy"
+     -- , testCase "RepeatLast test 26" $
+     --     snd (last (unwindMacros (bindInput [ ("a", "'xy'v")
+     --                                        , ("b", "'za'v")
+     --                                        , ("c", "'ab'v") ] coinput)
+     --                             (stringToKeyMacro "'c'v")))
+     --     @?= "xyyzxyyxyyzxyyxyyxyyzxyyxyyzxyyxyy"
+     -- , testCase "RepeatLast test 27" $
+     --     snd (last (unwindMacros (bindInput [ ("a", "'xy'V")
+     --                                        , ("b", "'za'v")
+     --                                        , ("c", "'ab'v") ] coinput)
+     --                             (stringToKeyMacro "'c'v")))
+     --     @?= "xyxyzxyxyxyxyzxyxyxyxyxyxyzxyxyxyxyzxyxyxyxy"
+     -- , testCase "RepeatLast test 28" $
+     --     snd (last (unwindMacros (bindInput [ ("a", "'xy'v")
+     --                                        , ("b", "'za'V")
+     --                                        , ("c", "'ab'v") ] coinput)
+     --                             (stringToKeyMacro "'c'v")))
+     --     @?= "xyyzxyyzxyyzxyyzxyyxyyzxyyzxyyzxyyzxyy"
+     -- , testCase "RepeatLast test 29" $
+     --     snd (last (unwindMacros (bindInput [ ("a", "'xy'v")
+     --                                        , ("b", "'za'V")
+     --                                        , ("c", "'ab'V") ] coinput)
+     --                             (stringToKeyMacro "'c'v")))
+     --     @?= "xyyzxyyzxyyxyyzxyyzxyyxyyzxyyzxyyxyyzxyyzxyy"
+     -- , testCase "RepeatLast test 30" $
+     --     snd (last (unwindMacros (bindInput [ ("a", "'xy'v")
+     --                                        , ("b", "'za'V")
+     --                                        , ("c", "'ab'V") ] coinput)
+     --                             (stringToKeyMacro "'c'V")))
+     --     @?= "xyyzxyyzxyyxyyzxyyzxyyxyyzxyyzxyyxyyzxyyzxyy"
+     -- , testCase "RepeatLast test 31" $
+     --     snd (last (unwindMacros (bindInput [ ("a", "'xy'v")
+     --                                        , ("b", "'za'v")
+     --                                        , ("c", "'ab'V") ] coinput)
+     --                             (stringToKeyMacro "'c'V")))
+     --     @?= "xyyzxyyxyyzxyyxyyzxyyxyyzxyy"
+     -- , testCase "RepeatLast test 32" $
+     --     snd (last (unwindMacros (bindInput [ ("a", "'xy'v")
+     --                                        , ("b", "'za'v") ] coinput)
+     --                             (stringToKeyMacro "'ab'vv")))
+     --     @?= "xyyzxyyxyyzxyyxyyxyyzxyyxyyzxyyxyy"
```

Comment:
And isn't this `xyyzxyyxyyzxyyxyyzxyyxyy`? :D

Reply:
Huh, indeed.

Reply:
I probably meant `'ab'vV`, but this test is fine, too.